source/StartConditions.h

```diff
@@ -15,32 +15,53 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
 #include "Account.h"
 #include "ConditionSet.h"
+#include "Conversation.h"
 #include "Date.h"
+#include "Ship.h"
 
 #include <list>
 
 class DataNode;
 class Planet;
-class Ship;
+class Sprite;
 class System;
 
 
 
 class StartConditions {
 public:
+	StartConditions() = default;
+	
+	// Implicit constructor
+	StartConditions(const DataNode &node);
+	
 	void Load(const DataNode &node);
 	// Finish loading the ship definitions.
 	void FinishLoading();
+	// Serialize the basic information of this start
+	void Save(DataWriter &out) const;
 	
 	Date GetDate() const;
-	// Get this start's planet, or New Boston if not set.
+	
 	const Planet &GetPlanet() const;
-	// Get this start's system, or Rutilicus if not set.
 	const System &GetSystem() const;
```

Comment:
These comments are technically still true. Might also be worth putting a comment above GetDate: `// Get this start's date, or 16/11/3013 if not set.`